client.sh

```diff
@@ -13,41 +13,49 @@ cat <<-END
 
 DESCRIPTION:
 
-		Script that creates LTSP client.
+    Script that creates LTSP client.
 
 USAGE:
 		
-		Simply run the script and enter the name 
-		of the client when prompted.
+    Simply run the script and enter the name 
+    of the client when prompted.
 						
 OPTIONS:
 
-		-h, --help
-		Display help
+    -h, --help
+    Display help
 
 END
 }
 
 create_client(){
 	
-	echo -n "Enter any name for LTSP client : "
-	read clientname
-	echo "Using ${INTERFACE}.."	
-	VBoxManage createvm --name "${clientname}" --register
-	cd "$HOME/VirtualBox VMs/${clientname}"
-	VBoxManage modifyvm "${clientname}" --memory 1024 --acpi on\
- --boot1 net --boot2 none --boot3 none --boot4 none --nic1 bridged\
- --bridgeadapter1 eno1 --nicpromisc1 allow-all
-	VBoxManage createhd --filename "$HOME/VirtualBox VMs/${clientname}/${clientname}.vdi" --size 1
-	VBoxManage storagectl "${clientname}" --name "IDE Controller" --add ide
-	VBoxManage startvm "${clientname}"
+    echo -n "Enter any name for LTSP client : "
+    read clientname
+    echo "Using ${INTERFACE}.."	
+    VBoxManage createvm --name "${clientname}" --register
+    cd "$HOME/VirtualBox VMs/${clientname}"
+    VBoxManage modifyvm "${clientname}" \
+        --memory 1024 \
```

Comment:
Can you add these options/settings to the command `createvm` (and don't use `modifyvm` at all)?

Reply:
I did spend some time trying to do this. But when looked at `VBoxManage` help option I saw that `createvm` offered limited functionality. Thats why I will need `modifyvm`
```
createvm                  --name <name>
                            [--groups <group>, ...]
                            [--ostype <ostype>]
                            [--register]
                            [--basefolder <path>]
                            [--uuid <uuid>]
```